docs/intro/cloud-providers/azure.mdx

```diff
@@ -46,12 +46,21 @@ az login
 
 When you're ready to deploy your application to Azure, you can use the Azure CLI Plugin to automate the deployment process.
 
-Use the `deploy` command from the root of your project to start an interactive deployment process, with prompts for Azure region, org. name and admin email:
+Use the `deploy:azure` command from the root of your project to start an interactive deployment process, with prompts for Azure region, org. name and admin email:
```

Comment:
We should probably provide information about what the Org name and admin email are used for. They're unique to Azure, so a link to the docs that explain what these values are for would be useful here.

Reply:
done